tests/gis_tests/layermap/models.py

```diff
@@ -92,3 +97,8 @@ class Invalid(models.Model):
                  'length': 'Length',
                  'path': 'LINESTRING',
                  }
+province_mapping = {'name': 'provincien',
```

Comment:
Please use the indentation style of  `co_mapping`. If someone were to fine an autoformatter that could fix the `inter_mapping` style throughout Django (it's rare), that would be nice.

Reply:
Fixed locally. Also aligned the other ones.

Reply:
If you do the unrelated alignments, please push them in a separate PR.